commons/src/main/java/org/infinispan/commons/api/functional/Param.java

```diff
@@ -44,6 +44,29 @@
    P get();
 
    /**
+    * Should this paramater go over the wire?
+    */
+   SerializeMethod serializeMethod();
```

Comment:
@ksobolew Don't worry about the mixup, as long as you keep those as separate commits, so you can easily rebase your changes. And thanks for the good work :)